src/wp-includes/block-supports/typography.php

```diff
@@ -452,6 +452,7 @@ function wp_get_computed_fluid_typography_value( $args = array() ) {
  * formula depending on available, valid values.
  *
  * @since 6.1.0
+ * @since 6.1.1 Adjusted min a
```

Comment:
Looks like this is an incomplete update. @ramonjd could you adjust this?

Reply:
Sorry, just saw this. Thanks @desrosj 

I see this PR has been merged. Should I open a new one to update the comment?

Reply:
Oh I see you updated it: https://github.com/ramonjd/wordpress-develop/blob/trunk/src/wp-includes/block-supports/typography.php#L455

Thank you very much!